src/component/container/page_list.tsx

```diff
@@ -34,13 +129,12 @@ export class PageList extends React.Component<PageListProps> {
 				open={this.pageListVisible}
 			>
 				{this.getProjectPages().map((page: PageRef, index) => (
-					<DropdownItem
-						name={page.getName()}
+					<PageListItem
+						id={page.getId()}
```

Comment:
it would be great if you could change this to `pageID` then it is clear what it is used for :)

Reply:
Done ✅